src/pair_wf.cpp

```diff
@@ -0,0 +1,403 @@
+/* ----------------------------------------------------------------------
```

Comment:
As a user-contributed pair style, please move it to the USER-MISC package and update the corresponding README file. A reference to this pair_style should be added to `doc/src/Commands_pair.rst` and `doc/src/pair_style.rst`
Please see the summary output at the end of "make html" (in the doc folder). Also please address any spelling issues indicated by "make spelling" (in the doc folder).

Reply:
Updated these files